src/database/DataTypes/StocktakeItem.js

```diff
@@ -41,7 +41,7 @@ export class StocktakeItem extends Realm.Object {
    * @return  {boolean} True if StocktakeBatches have adjustments
    */
   get hasBatchWithQuantityChange() {
-    return this.batches.some(stocktakeBatch => stocktakeBatch.difference !== 0);
+    return this.batches.some(stocktakeBatch => stocktakeBatch.hasBeenCounted);
```

Comment:
Would be cleaner to remove this getter all together and replace references with `hasCountedBatches` below seeing as they are now functionally the same.

Reply:
@Chris-Petty -> nice spotting, adjusted.